Packages/vcs/Lib/editors/fillarea.py

```diff
@@ -0,0 +1,258 @@
+from vcs import vtk_ui
+from vcs.vtk_ui import behaviors
+from vcs.colorpicker import ColorPicker
+import priority
+
```

Comment:
Would be nice to have some text describing the class. Without it, it is hard to understand what's going on.